packaging/plugins/provision.sh

```diff
@@ -14,50 +16,39 @@ function print_plugins_params() {
     done
 }
 
-function install_dependencies(){
+function install_windows_dependencies(){
+    # Prepare virtualenv in windows machine
     echo "## Installing necessary dependencies"
+    pip install virtualenv
```

Comment:
please pin to a version, so it will be stable

Reply:
Done